src/main/java/com/ays/user/model/dto/request/UserSupportStatusUpdateRequest.java

```diff
@@ -0,0 +1,46 @@
+package com.ays.user.model.dto.request;
+
+import com.ays.common.util.validation.EnumValidation;
+import com.ays.user.model.enums.UserSupportStatus;
+import jakarta.validation.constraints.AssertTrue;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.EnumSet;
+
+/**
+ * Request object for updating the support status of a user.
+ */
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserSupportStatusUpdateRequest {
+
+    @NotNull
+    private UserSupportStatus supportStatus;
+
+    @AssertTrue(message = "IS SUPPORT STATUS NOT VALID")
```

Comment:
Buradaki mesajı şu şekilde güncellememiz daha iyi olabilir, burada soru gibi olmuş;
`SUPPORT STATUS IS NOT ACCEPTED!`

Reply:
Ok. Bu kısmı revise ettim.